integration/cri-o/cri-o.sh

```diff
@@ -10,6 +10,7 @@ set -e
 SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
 source "${SCRIPT_PATH}/../../.ci/lib.sh"
 source "${SCRIPT_PATH}/crio_skip_tests.sh"
+source /etc/os-release
```

Comment:
just a note - at some future point I suspect this will fail, as not all distros supply that file.

Reply:
Good point. It might be a good idea if we put that logic into `.ci/lib.sh` so that once you've sourced that file, you get access to `VERSION_ID`, etc.